tests/helm/testdata/goldenfile/setup/setup/basic.output.yaml

```diff
@@ -33,7 +33,7 @@ spec:
             defaultMode: 0777
       containers:
         - name: setup
-          image: public.ecr.aws/sumologic/kubernetes-setup:3.8.0
+          image: public.ecr.aws/sumologic/kubernetes-setup:%CURRENT_CHART_VERSION%
```

Comment:
I think it's only a conincidence that the current setup image version `3.9.0` is equal to current chart version `3.9.0`, right? Which means we should still use hardcoded value `3.9.0` here instead of the current chart version replacement string.

Reply:
```
-       (string) (len=5) "image": (string) (len=47) "public.ecr.aws/sumologic/kubernetes-setup:3.9.0",
+       (string) (len=5) "image": (string) (len=65) "public.ecr.aws/sumologic/kubernetes-setup:%CURRENT_CHART_VERSION%"
```
when the test runs the actualObject seems to have `public.ecr.aws/sumologic/kubernetes-setup:%CURRENT_CHART_VERSION%` which doesn't match `public.ecr.aws/sumologic/kubernetes-setup:3.9.0`
causing the test to fail

Reply:
@astencel-sumo @swiatekm-sumo I regenerated this file using `make regenerate-goldenfiles` and it substituted the version with `%CURRENT_CHART_VERSION%`

Reply:
The regenerate command isn't very smart and can get confused at edge cases. You need to manually fix its mistakes at times.

Reply:
ok, it looks like the test does fail when the expected is hardcoded to `3.9.0`. Does the `basic.input.yaml` need to be changed?

Reply:
not replacing the k8s-setup chart version using a regex, seems to fix this issue